src/__tests__/Batcher.ts

```diff
@@ -298,7 +298,7 @@ describe("Batcher", () => {
       }
       events.splice(0, 2);
       
-      expect(setItemsSpy).toHaveBeenCalledTimes(1);
+      expect(setItemsSpy).toHaveBeenCalledTimes(2);
```

Comment:
This has been broken since (at least) the 1.0.3 release but I'm not sure if this "fix" is apprioriate

Reply:
CC @alex-tpom6oh